### DISCUSSION TITLE
Missing glances.conf

### DISCUSSION THREAD

**Discussion by atcapollo** [nicolargo/glances#2484] [General]
Installed glances via pip on Ubuntu 22.04 but cannot find glances.conf

**To Reproduce**
1. pip install --user 'glances[all]'
2. No default glances.conf in either ~/.config/glances or /etc/glances
3. Have done a find / -name "glances.conf" but nothing on system

**Expected behavior**
glances.conf should be somewhere?!

**Comment by Swipe05** [comment]
Hello atcapollo I need to contact you about fail2ban + ipsec please chawopvp@gmail.com

**Comment by nicolargo** [comment]
The path to the glances.conf file depend of your system/installation. It is displayed in the Glances log file when Glances starts:

```
2023-07-01 11:03:17,198 -- INFO -- Start Glances 3.4.0.3
2023-07-01 11:03:17,198 -- INFO -- CPython 3.10.6 (/usr/bin/python3) and psutil 5.9.1 detected
2023-07-01 11:03:17,207 -- INFO -- Read configuration file '/home/nicolargo/.config/glances/glances.conf'
...
```

To get the path to the Glances log file please use:

```
# glances --version
Glances v3.4.0.3 with PsUtil v5.9.1
Log file: /home/nicolargo/.local/share/glances/glances.log
```
